packages/react/src/components/Accordion/__tests__/Accordion-test.js

```diff
@@ -27,4 +28,27 @@ describe('Accordion', () => {
 
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should have no AVT1 violations', async () => {
+    const mountNode = document.createElement('div');
+    document.body.appendChild(mountNode);
+
+    render(
+      <Accordion className="extra-class">
+        <AccordionItem className="child" title="Heading A">
+          Panel A
+        </AccordionItem>
+        <AccordionItem className="child" title="Heading B">
+          Panel B
+        </AccordionItem>
+        <AccordionItem className="child" title="Heading C">
+          Panel C
+        </AccordionItem>
+      </Accordion>,
+      mountNode
+    );
+
+    await expect(document).toHaveNoViolations();
+    mountNode.parentNode.removeChild(mountNode);
```

Comment:
Does this code run when the assertion fails?

Reply:
Good call! Let me pull it out into afterEach 👍